Other/MergeShops/BloodMoonVampireMerge.cs

```diff
@@ -0,0 +1,106 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreFarms.cs
+//cs_include Scripts/CoreStory.cs
+//cs_include Scripts/CoreAdvanced.cs
+//cs_include Scripts/Story/BloodMoon.cs
+using RBot;
+using RBot.Items;
+using RBot.Options;
+
+public class BloodMoonVampireMergeTemp
+{
+    public ScriptInterface Bot => ScriptInterface.Instance;
+    public CoreBots Core => CoreBots.Instance;
+    public CoreFarms Farm = new();
+    public CoreStory Story = new();
+    public CoreAdvanced Adv = new();
+    public static CoreAdvanced sAdv = new();
+
+    public List<IOption> Generic = sAdv.MergeOptions;
+    public string[] MultiOptions = { "Generic", "Select" };
+    public string OptionsStorage = sAdv.OptionsStorage;
+    // [Can Change] This should only be changed by the author.
+    //              If true, it will not stop the script if the default case triggers and the user chose to only get mats
+    private bool dontStopMissingIng = false;
+
+    public BloodMoon BloodMoonQuests = new BloodMoon();
+
+    public void ScriptMain(ScriptInterface bot)
+    {
+        Core.SetOptions();
+
+        BuyAllMerge();
+
+        Core.SetOptions(false);
+    }
+
+    public void BuyAllMerge()
+    {
+
+        if (!Core.isCompletedBefore(6067))
```

Comment:
^_^ the if bit isnt needed as its already in the story script itself :D all you need is the "BloodMoonQuests.BloodMoonSaga();"

Reply:
done, has already been removed